package.json

```
@@ -21,7 +21,7 @@
   "dependencies": {
     "bson": "^6.7.0",
```

```suggestion
    "bson": "^6.10.0",
```

[mongodb 6.11.0](https://github.com/mongodb/node-mongodb-native/blob/main/HISTORY.md#6110-2024-11-22) uses [bson 6.10.0](https://github.com/mongodb/node-mongodb-native/blob/122421d8f4d3da661c1aded288138690ab759714/package.json#L29)